### PR TITLE
Reveal block

### PR DESCRIPTION
**Pull request by RobLoweMOJ**
Also moved a few hero styles to be after tablet breakpoint as it was broken on mobile

### CODE REVIEW

File: src/09-reveal/style.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+**********************************************************************************************/
+
+/**
+* This file contains imported styling to support GOV UK Frontend usage in the plugin
+*
+* To import styles from govuk-frontend, simply include the Gov UK module using
+*  @imports with a URL reference
+**/
+
+// https://design-system.service.gov.uk/components/accordion/
```
Comment (brown-a2) [inline src/09-reveal/style.scss:16]: Might want to swap this for detail component?

File: src/frontend.js [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
 // import dependencies
 import { Accordion } from 'govuk-frontend';
+import { Details } from 'govuk-frontend';
 
 // get components
 /** Accordion **/
```
Comment (brown-a2) [inline src/frontend.js:11]: Do you need to add Details initialization below as well?

File: src/09-reveal/editor.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  display: block;
+  width: 0;
+  height: 0;
+  -webkit-clip-path: polygon(0 0,50% 100%,100% 0);
+  clip-path: polygon(0 0,50% 100%,100% 0);
+  border-color: transparent;
+  border-style: solid;
+  border-width: 12.124px 7px 0;
+  border-top-color: inherit;
+}
```
Comment (wilson1000-MoJ) [inline src/09-reveal/editor.scss:22]: Morning :) I posted about EditorConfig to assist in reaching our minimum standards in wordpress-gang earlier this week. Did you see this?

In fact, this stylesheet is formatted differently to the one above (hero/style.scss) - which standard are you using?

File: src/09-reveal/style.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+@import "node_modules/govuk-frontend/govuk/components/details/details";
```
Comment (wilson1000-MoJ) [inline src/09-reveal/style.scss:17]: Just a heads up. Have you tested the accordion component with this integration? Also, the loading order of components matters so you may need to put this in the main style.scss file.

As far as I know styling in the accordion isn't compatible with some of the components in govuk-frontend so please beware.

Comment (RobLoweMOJ) [reply src/09-reveal/style.scss]: Seems to be working with accordion.

File: src/frontend.js (hunk 2 above)
Comment (RobLoweMOJ) [reply src/frontend.js:11]: Done

File: src/09-reveal/style.scss (hunk 1 above)
Comment (RobLoweMOJ) [reply src/09-reveal/style.scss]: Done

File: src/09-reveal/editor.scss (hunk 3 above)
Comment (RobLoweMOJ) [reply src/09-reveal/editor.scss]: Styles format updated.

File: src/09-reveal/style.scss (hunk 4 above)
Comment (wilson1000-MoJ) [reply src/09-reveal/style.scss]: Ok cool, we just need to make sure we test it each time we import a govuk component, and for sure if we import the button component, I'm quite sure this is what broke the accordion styling before.

**Pull request merged**